docs/app/references/experiments.mdx

```diff
@@ -257,63 +257,10 @@ creating `e2e` and `component` objects inside your Cypress configuration.
-| `experimentalRunAllSpecs`         | `false` | Enables the "Run All Specs" UI feature, allowing the execution of multiple specs sequentially.             |
-| `experimentalOriginDependencies`  | `false` | Enables support for `Cypress.require` within `cy.origin`.                                                  |
-| `experimentalSkipDomainInjection` | `null`  | Removes injecting `document.domain` into `text/html` pages for any sites that match the provided patterns. |
-
-#### Experimental Skip Domain Injection
-
-Under the hood, Cypress
-[injects document.domain](/app/guides/cross-origin-testing#What-Cypress-does-under-the-hood)
-into your test application to lessen the burden of navigation. This is well
-described in our [Cross Origin Testing](/app/guides/cross-origin-testing)
-guide. However, some sites have compatibility issues with this feature.
-
-The `experimentalSkipDomainInjection` option disables injecting
-`document.domain` inside Cypress. When enabled, all cross-origin/subdomain
-navigation must use `cy.origin()`, which may make tests a bit more verbose. We
-only recommend including your site pattern if you are having issues running
-Cypress out of the box and suspect setting `document.domain` is interfering with
-your site's ability to render properly.
-
-Before enabling, verify your application is not implementing frame busting
-techniques, which you can mitigate with the
-[`modifyObstructiveCode`](/app/references/configuration#modifyObstructiveCode)
-and
-[`experimentalModifyObstructiveThirdPartyCode`](/app/guides/cross-origin-testing#Modifying-Obstructive-Third-Party-Code)
-flags.
-
-At this point in time, we are aware of the following sites that require the
-`experimentalSkipDomainInjection` option to be set to be tested properly:
-
-- Google
-- Salesforce
-
-This flag can be enabled by passing an array of origin URLs or
-[minimatch](https://github.com/isaacs/minimatch) glob patterns:
-
-:::cypress-config-example
-
-```js
-{
-  e2e: {
-    experimentalSkipDomainInjection: [
-      '*.salesforce.com',
-      '*.force.com',
-      '*.google.com',
-    ]
-  }
-}
-```
-
-:::
-
-If using other Salesforce domains, such as
-[enhanced domains](https://help.salesforce.com/s/articleView?id=sf.domain_name_enhanced.htm&type=5),
-you will need to add the correct matching glob pattern.
+| Option                    | Default | Description                                                                                               |
+| ------------------------- | ------- | --------------------------------------------------------------------------------------------------------- |
+| `experimentalStudio`      | `false` | Generate and save commands directly to your test suite by interacting with your app as an end user would. |
+| `experimentalRunAllSpecs` | `false` | Enables the "Run All Specs" UI feature, allowing the execution of multiple specs sequentially.            |
```

Comment:
@cacieprins Did you mean to remove `experimentalOriginDependencies` here?

Reply:
Accidental - resolved in https://github.com/cypress-io/cypress-documentation/pull/5974/commits/644aa612b9a136472c6f1be955f49458f3770b70